locust/README.md

```diff
@@ -0,0 +1,41 @@
+## Purpose 
+This directory contains a Dockerfile that builds `Locust` using OpenSSL v3 using the [OQS provider](https://github.com/open-quantum-safe/oqs-provider) and Python3, which allows `Locust` to negotiate quantum-safe keys and use quantum-safe authentication in TLS 1.3.
+
+## Quick start
+
+1) Be sure to have [docker installed](https://docs.docker.com/install). 
+2) Run `docker build -t oqs-locust:0.0.1 .` to create a post quantum-enabled Locust docker image.
+3) In order to configure endpoints and their weight, modify the file [scerarios/locusfile.py](scenarios/locustfile.py), more information can be found in [USAGE.md](USAGE.md)
+4) To verify all components perform quantum-safe operations, first start the container with docker compose 
+
+```LOGGER_LEVEL=DEBUG HOST=https://qsc-nginx.discovery.hi.inet:4433 docker compose  up --scale master=1 --scale worker=8```
```

Comment:
I think adding newlines before and after the command (inside the ```) tags will help with the display.

Reply:
I agree, fix in this [commit](https://github.com/open-quantum-safe/oqs-demos/pull/304/commits/91769ca63c25a4b9fb393634cf6ae2397f87d132)